src/components/Class/ClassesList/index.js

```diff
@@ -0,0 +1,65 @@
+import React, { useEffect, useState } from "react";
+import Axios from "axios";
+import "./styles.scss";
+
+function ClassList() {
+  const [allClass, setAllClass] = useState([]);
+
+  const convertTime = (timeString) => {
+    var hourEnd = timeString.indexOf(":");
+    var H = +timeString.substr(0, hourEnd);
+    var h = H % 12 || 12;
+    var ampm = H < 12 ? "AM" : "PM";
+    timeString = h + timeString.substr(hourEnd, 3) + ampm;
+    return timeString;
+  };
+
+  useEffect((e) => {
+    const token =
+      "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IjE5NiIsImVtYWlsIjoiaWFtc2FxdWlibmFzaW1AZ21haWwuY29tIiwiaWF0IjoxNjAwNTM1MzMxLCJleHAiOjE2MzIwOTI5MzF9.vuSZRabkDGa5kdx51D7K9R7lzZL3sElBUTnHs2x-GDY";
+    Axios.get("http://dev-api.navgurukul.org/classes/all", {
+      headers: { Authorization: `Bearer ${token}` },
+    }).then((res) => {
+      setAllClass(res.data);
+    });
+  }, []);
+
+  return (
+    <>
+      <table>
+        <thead>
+          <tr>
+            <th>Facilitator Name</th>
+            <th>Title</th>
+            <th>Description</th>
+            <th>Language</th>
+            <th>Class type</th>
+            <th>Date</th>
+            <th> Start Time </th>
+            <th>End Time</th>
+          </tr>
+        </thead>
+        {allClass.map((item, index) => {
+          return (
```

Comment:
Could we move to its separate component? The header is fine. the item/class I think we can move into different component.